processor/resourcedetectionprocessor/internal/gcp/documentation.md

```diff
@@ -21,6 +21,9 @@
 | gcp.cloud_run.job.task_index | The Job execution task index | Any Str | true |
 | gcp.gce.instance.hostname | The hostname of the GCE instance. | Any Str | false |
 | gcp.gce.instance.name | The name of the GCE instance. | Any Str | false |
+| gcp.gce.instance_group_manager.name | The name of an instanceGroupManager. | Any Str | true |
+| gcp.gce.instance_group_manager.region | The region of a regional instanceGroupManager. | Any Str | true |
+| gcp.gce.instance_group_manager.zone | The zone of a zonal instanceGroupManager. | Any Str | true |
```

Comment:
Should we use availability_zone instead of zone for consistency with cloud.availability_zone?  I can see wanting to keep it shorter.  Any other reasons why you went with zone?

Reply:
I went with `zone` because this is a GCP-specific attribute name and `zone` is the terminology used in GCP.